### ISSUE TITLE
Spotify Update Shuts Down Skin

### ISSUE THREAD

**Issue by seokster** [devinhalladay/spotio#37]
I've installed Spotio but the auto update on Spotify keeps getting rid of the skin. I've tried methods of stopping Spotify's auto update but to no avail.

**Comment by alexslee** [comment]
Yeah, I'm not even sure if it's auto-update at this point, as I receive the message to restart to actually perform the update after Spotio force-closes (which happens maybe < 30 seconds into the first open). Tried patching with a script to prevent auto-updates, which stopped the message from appearing, but the app still crashed and re-opened to the original dark color scheme.

**Comment by markfried96** [comment]
Is there any chance to get the Skin work again? I really like the design of the Spotio App. I hope there will be any solution very soon.

**Comment by devinhalladay** [comment]
Hey everyone —

Spotify does auto-update; they broke the vulnerability I was using to inject Spotio no more than a week after Spotio was originally released. I still don't have the time to get Spotio up and running again (it is doable, and I have a lot of good ideas for what to do with it once it's working again), though.

If any of you have spare time and want tot take a crack at re-injecting Spotio reliably through some script or installer, that's the first major roadblock to getting Spotio up and running again.

**Comment by jnylen** [comment]
FYI,
Spotio still works with arch. I'm running it now.

**Comment by kiosion** [comment]
I was having the same issue - and I partially fixed it. I used this [script](https://www.quora.com/How-can-I-prevent-Spotify-from-auto-updating-on-Mac) to try and fix it, then I went into "Get Info" (I'm using Mac OSX) and Locked the app. Now it crashes every 30 seconds or so, but it doesn't revert to the old spotify skin.

I love this project and would really like to have a working version - It looks amazing.

**Comment by devinhalladay** [comment]
I'd still like to get this project working again, but definitely don't have the time to do this myself.

If any of you technical users who like Spotio and want to see it working again would like to submit a pull request to help fix this issue, that would be extremely appreciated.
